batcher/aligned/src/main.rs

```diff
@@ -307,7 +307,16 @@ async fn main() -> Result<(), AlignedError> {
                     .map_err(|e| SubmitError::GenericError(e.to_string()))?
             } else {
                 warn!("Missing keystore used for payment. This proof will not be included if sent to Eth Mainnet");
-                LocalWallet::from_str(ANVIL_PRIVATE_KEY).expect("Failed to create wallet")
+                match LocalWallet::from_str(ANVIL_PRIVATE_KEY) {
+                    Ok(wallet) => wallet,
+                    Err(e) => {
+                        warn!(
+                            "Failed to create wallet from anvil private key: {}",
+                            e.to_string()
+                        );
+                        return Ok(());
```

Comment:
Should the error be forwarded?

Reply:
This is already the return of the main function. Not much to do, but we could create an issue to return an Error so the application ends with an Error